src/services.js

```diff
@@ -384,6 +369,19 @@ export class Services {
     ));
   }
 
+  /**
+   * Not installed by default; must be installed in extension code before use.
+   * @param {!Element|!ShadowRoot} element
+   * @return {!./service/position-observer/position-observer-impl.PositionObserver}
+   * @throws If the service is not installed.
+   */
+  static positionObserverForDoc(element) {
+    return /** @type {!./service/position-observer/position-observer-impl.PositionObserver} */ (getServiceForDoc(
+      element,
+      'position-observer'
+    ));
+  }
```

Comment:
@zhouyx PTAL at PositionObserver usages above.

Reply:
sorry, missed the comment here. LGTM